### DISCUSSION TITLE
Share App Feature

### DISCUSSION THREAD

**Discussion by surendhar1992** [microsoft/powerplatform-build-tools#248] [General]
Share App Feature . Once the App is deployed using service connection ,Users are not able to see the app other than Power App admin for the first time admin need to share the app and make co-owner. If there is a tasks to share app with co-owner or user permissions that would be really great.

**Comment by davidjenni** [comment]
@surendhar1992 the capability to assign users/owners/groups to a app/solution was recently added to the pac CLI itself, and is in the process of becoming available as a PP-BT task as well in one of our next monthly refreshes. See also #233 and #238

**Reply by surendhar1992** [reply]
Thank you.

**Reply by INC02203** [reply]
ETA for this feature release?

**Comment by petrochuk** [comment]
@INC02203
> ETA for this feature release?

This is open-source repo with source code for all tasks published including the ones David mentioned:
https://github.com/microsoft/powerplatform-build-tools/tree/main/src/tasks/assign-user/assign-user-v2
https://github.com/microsoft/powerplatform-build-tools/tree/main/src/tasks/assign-group/assign-group-v2

You can even submit pull request of new tasks or modify existing ones
